### DISCUSSION TITLE
Is it possible to set a total limit for multiple inverters?

### DISCUSSION THREAD

**Discussion by byOgram** [tbnobody/OpenDTU#774] [Q&A]
Hi, I'm wondering if there's a function that allows setting a maximum limit for multiple inverters when they are connected. This means that the DTU decides which inverter can feed how much power as long as they always stay below the set total value. Does this function already exist? If so, where can I find it? Or would it be useful to implement it? 

For example, let's say I have three inverters with a maximum output of 600 watts each, and I want to limit the total output to 1 kW. Is there a function that allows me to do this? If yes, could you let me know where to find it? If not, do you think it would be useful to have such a function?

For instance, if Inverter 1 produces 400 watts, the DTU may allow Inverter 2 to produce 300 watts and Inverter 3 to produce 300 watts, for a total power output of 1 kW (1000 watts). Or, if Inverter 1 produces only 100 watts, the DTU may allow Inverter 2 to produce 500 watts and Inverter 3 to produce 400 watts.

This way, you can ensure that the total power output from all inverters never exceeds the limit you set, and the DTU can balance the power output between the inverters based on their capabilities and the current system demand.

Regards.

**Comment by Solarteur** [comment]
A limitation function for the sum of several inverters does not yet exist. I don't know Thomas' plans, from my point of view OpenDTU should remain the bridge between the inverter(s) and a WLAN.
I.e. your suggestions should be implemented on a home automation system or another device that reads the mqtt values.
